lib/svn2git/migration.rb

```diff
@@ -154,6 +154,7 @@ def parse(args)
 
     def self.escape_quotes(str)
       str.gsub(/'|"/) { |c| "\\#{c}" }
+         .gsub(/\$/)  { |c| '\\$' }
```

Comment:
I believe we've had to remove it in the past because it's not implemented in Ruby 1.8. A large number of svn2git users don't otherwise manage Ruby installations, so they just install on whatever their system already has. We support at least the Ubuntu LTS releases (so, going back 5 years) and a couple versions of MacOS X.

I'd need to reevaluate since Ubuntu 10.04 has now fallen off the support path, but we might not be able to realistically use any 1.9+ specific methods yet.